radiation/python/radlw/taumol_gt4py.py

```diff
@@ -0,0 +1,4957 @@
+    id110: FIELD_INT,
+    id200: FIELD_INT,
+    id210: FIELD_INT,
+    id001: FIELD_INT,
+    id011: FIELD_INT,
+    id101: FIELD_INT,
+    id111: FIELD_INT,
+    id201: FIELD_INT,
+    id211: FIELD_INT,
+):
+    from __externals__ import nspa, nspb, laytrop, ng03, nlay, ns03, oneminus
+
+    with computation(PARALLEL), interval(0, laytrop):
+        #  --- ...  minor gas mapping levels:
+        #     lower - n2o, p = 706.272 mbar, t = 278.94 k
+        #     upper - n2o, p = 95.58 mbar, t = 215.7 k
+
+        refrat_planck_a = chi_mls[0, 0, 0][0, 8] / chi_mls[0, 0, 0][1, 8]
+        refrat_planck_b = chi_mls[0, 0, 0][0, 12] / chi_mls[0, 0, 0][1, 12]
+        refrat_m_a = chi_mls[0, 0, 0][0, 2] / chi_mls[0, 0, 0][1, 2]
+        refrat_m_b = chi_mls[0, 0, 0][0, 12] / chi_mls[0, 0, 0][1, 12]
+
+        speccomb = colamt[0, 0, 0][0] + rfrate[0, 0, 0][0, 0] * colamt[0, 0, 0][1]
+        specparm = colamt[0, 0, 0][0] / speccomb
+        specmult = 8.0 * min(specparm, oneminus)
+        js = 1 + specmult
+        fs = mod(specmult, 1.0)
+        ind0 = ((jp - 1) * 5 + (jt - 1)) * nspa + js - 1
+
+        speccomb1 = colamt[0, 0, 0][0] + rfrate[0, 0, 0][0, 1] * colamt[0, 0, 0][1]
+        specparm1 = colamt[0, 0, 0][0] / speccomb1
+        specmult1 = 8.0 * min(specparm1, oneminus)
+        js1 = 1 + specmult1
+        fs1 = mod(specmult1, 1.0)
+        ind1 = (jp * 5 + (jt1 - 1)) * nspa + js1 - 1
+
+        speccomb_mn2o = colamt[0, 0, 0][0] + refrat_m_a * colamt[0, 0, 0][1]
+        specparm_mn2o = colamt[0, 0, 0][0] / speccomb_mn2o
+        specmult_mn2o = 8.0 * min(specparm_mn2o, oneminus)
+        jmn2o = 1 + specmult_mn2o - 1
+        fmn2o = mod(specmult_mn2o, 1.0)
+
+        speccomb_planck = colamt[0, 0, 0][0] + refrat_planck_a * colamt[0, 0, 0][1]
+        specparm_planck = colamt[0, 0, 0][0] / speccomb_planck
+        specmult_planck = 8.0 * min(specparm_planck, oneminus)
+        jpl = 1 + specmult_planck - 1
+        fpl = mod(specmult_planck, 1.0)
+
+        inds = indself - 1
+        indf = indfor - 1
+        indm = indminor - 1
+        indsp = inds + 1
+        indfp = indf + 1
+        indmp = indm + 1
+        jmn2op = jmn2o + 1
+        jplp = jpl + 1
+
+        #  --- ...  in atmospheres where the amount of n2O is too great to be considered
+        #           a minor species, adjust the column amount of n2O by an empirical factor
+        #           to obtain the proper contribution.
+
+        p = coldry * chi_mls[0, 0, 0][3, jp]
+        ratn2o = colamt[0, 0, 0][3] / p
+        if ratn2o > 1.5:
+            adjfac = 0.5 + (ratn2o - 0.5) ** 0.65
+            adjcoln2o = adjfac * p
+        else:
+            adjcoln2o = colamt[0, 0, 0][3]
+
+        id000 = id000
+        id010 = id010
+        id100 = id100
+        id110 = id110
+        id200 = id200
+        id210 = id210
+
+        id001 = id001
+        id011 = id011
+        id101 = id101
+        id111 = id111
+        id201 = id201
+        id211 = id211
```

Comment:
Could add a comment that this is a workaround for a known gt4py bug, should be able to be removed in the next tag. But if it's in your plan to do that when the new tag is available, then a note might be redundant and unecessary.

Reply:
Yes, will do. Have done that in the cldprop pull request too